bin/setenv-docker-customize.sh

```diff
@@ -717,6 +717,55 @@ else
   touch /opt/exo/_done.patches
 fi
 
+# -----------------------------------------------------------------------------
+# Import self-signed certificates to Keystore
+# -----------------------------------------------------------------------------
+if [ -f /opt/exo/_done.ssca ]; then
+  echo "INFO: self-signed certificates keystore import already done! skipping this step."
+else
+  # self-signed certificates authorization
+  if [ -z "${EXO_SELFSIGNEDCERTS_HOSTS:-}" ]; then
+    echo "# no self-signed certificate to be imported from EXO_SELFSIGNEDCERTS_HOSTS environment variable."
+  else
+    echo "# Cloning JDK cacerts keystore to custom one to be used for self-signed certificates import..."
+    rm -rf /opt/exo/custkeystore
+    mkdir -p /opt/exo/custkeystore
+    _cacertsPath=/opt/exo/custkeystore/cacerts
```

Comment:
It will exclude the jdk certs. No side effect?

Reply:
No, it will copy the JDK keystore to another custom (root permission required for keytool). Then keytool will import custom certs.